main.go

```diff
@@ -22,8 +26,18 @@ const (
 	envVarOrchestrionLogFile  = "ORCHESTRION_LOG_FILE"
 	envVarOrchestrionLogLevel = "ORCHESTRION_LOG_LEVEL"
 	envVarToolexecImportPath  = "TOOLEXEC_IMPORTPATH"
+	envVarCPUProfiling        = "CPU_PROFILING"
+	envVarHeapProfiling       = "HEAP_PROFILING"
+	envVarExecutionTracing    = "EXECUTION_TRACING"
+	envVarProfilePrefix       = "PROFILE_PREFIX"
 )
 
+var profilePrefix string
+
+type ctxKeyCPUProfiling struct{}
+type ctxKeyHeapProfiling struct{}
+type ctxKeyExecutionTracing struct{}
```

Comment:
I find it a little awkward that the `profilePrefix` is shipped into a package-local variable; while the three enablement variables are stored as `context.Context` values.

I think it should be possible to:
- Have all flags use a `Destination` that is a variable local to `main`
- Start all the requested profilers in a `Before` closure

Reply:
I got rid of the variable for the profilePrefix, and moved all the profile initialization into `Before` as suggested.